contracts/BIFI/interfaces/common/IUniswapRouter.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity ^0.6.0;
+pragma solidity >=0.6.0 <0.9.0;
```

Comment:
If `interfaces/common/IUniswapRouterETH.sol` works, consider using that rather than changing `IUniswapRouter.sol`.